### PR TITLE
Fix #13 and #14.

### LINKED ISSUE #13: Drop support for JAX-RS
It's not clear what value the plain JAX-RS programming model provides, hence I would suggest to drop the JAX-RS support as per discussion in https://groups.google.com/forum/#!topic/microprofile/IbB2TziPWLU

### PR DESCRIPTION
**Pull request by heiko-braun**
Summary:

Removed all JAX-RS related API classes and corresponding tests. Updated the tests to support several commonly expected scenarios.

These changes have been discussed and agreed on the call at the 27.06.17

### CODE REVIEW

File: tck/src/main/java/org/eclipse/microprofile/health/tck/MultipleProceduresFailedTest.java [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+        return deployment;
+    }
+
+    /**
+     * Verifies the health integration with CDI at the scope of a server runtime
+     */
+    @Test
+    @RunAsClient
+    public void testFailureResponsePayload() throws Exception {
+        Response response = getUrlContents("http://localhost:8080/health");
```
Comment (Emily-Jiang) [inline tck/src/main/java/org/eclipse/microprofile/health/tck/MultipleProceduresFailedTest.java]: Is this url fixed? Can the appname be appended on the url? I think it just means the ending bit is /health

Comment (heiko-braun) [inline tck/src/main/java/org/eclipse/microprofile/health/tck/MultipleProceduresFailedTest.java]: I can think of both cases: a) appcontext + /health or b) a top level /health. Maybe even c) runtime chosen context + a) or b)

Comment (heiko-braun) [inline tck/src/main/java/org/eclipse/microprofile/health/tck/MultipleProceduresFailedTest.java]: The most simple and implementation independent would be b) i guess

Comment (keilw) [inline tck/src/main/java/org/eclipse/microprofile/health/tck/MultipleProceduresFailedTest.java]: Is there still a URL with REST/HTTP removed?

Comment (Emily-Jiang) [inline tck/src/main/java/org/eclipse/microprofile/health/tck/MultipleProceduresFailedTest.java]: I think a) is useful @heiko-braun ! For b), what should be happening when more than one apps running and listening on the same port?

Comment (heiko-braun) [inline tck/src/main/java/org/eclipse/microprofile/health/tck/MultipleProceduresFailedTest.java]: @keilw Not sure I understand the question. Can you elaborate?

Comment (heiko-braun) [inline tck/src/main/java/org/eclipse/microprofile/health/tck/MultipleProceduresFailedTest.java]: @Emily-Jiang right, that's something I havn't thought about. I've created https://github.com/eclipse/microprofile-health/issues/19 to address this.

Comment (keilw) [inline tck/src/main/java/org/eclipse/microprofile/health/tck/MultipleProceduresFailedTest.java]: @heiko-braun @Emily-Jiang Ok you just refer to a small portion of this PR that's only TCK related.

**Pull request merged**